packages/macro/index.d.ts

```diff
@@ -172,6 +172,7 @@ export type TransProps = {
   comment?: string
   values?: Record<string, unknown>
   context?: string
+  children?: string | any[] | React.ReactNode
```

Comment:
Shouldn't that just be  

```ts
children?: ReactNode | undefined
```

That was the previous definition.  The `string` part is already part of `ReactNode`, and the `any[]` seems too wide.

Reply:
That is right, have fixed it now 😄